libraries/iio/iio_types.h

```diff
@@ -73,8 +73,7 @@ enum iio_chan_type {
 	IIO_ALTVOLTAGE,
 	IIO_ANGL_VEL,
 	IIO_TEMP,
-	/* All new types must be added before this field */
-	IIO_LAST_TYPE
+	IIO_CAPACITANCE
```

Comment:
Wasn't IIO_LAST_TYPE there for a reason? Also, i would move this change to a separate commit

Reply:
no reason that I could see, completely unused.